packages/breadcrumb/src/react/__specs__/index.spec.js

```diff
@@ -21,7 +21,7 @@ test('click on disabled button with href does not trigger onClick', () => {
     callCount += 1
   }
   const breadcrumb = mount(
-    <Breadcrumb disabled onClick={onClick} href="https://foo.com">
+    <Breadcrumb disabled onClick={onClick} href='https://foo.com'>
```

Comment:
Double quotes in jsx has been our style convention thus far.  Should we keep it that way?

Reply:
i'll update eslint to use double quotes in JSX